### DISCUSSION TITLE
Grunt error when installing local/dev version

### DISCUSSION THREAD

**Discussion by funkywaddle** [Leantime/leantime#792] [General] [answered]
I am trying to install a local/dev version using the github clone and following the instructions [here](https://docs.leantime.io/#/installation/quick-installation?id=installation-development).

I am receiving an error when running the grunt command: `grunt Build-All`

As I am a dedicated back-end developer and not a front-end dev, it's likely that my environment isn't setup right, so I apologize before-hand if this is the case.

You can see the output of the grunt command [here](https://projects.qualitycoder.net/images/grunt_build_all.png)

**Comment by marcelfolaron** [comment, marked as answer]
Odd, I’ve never seen this issue before but seems a module is missing. Did a quick search: can you try to follow the suggestions here: 
https://stackoverflow.com/questions/53408846/cannot-run-grunt-command-because-of-error-cannot-find-module-liftoff

**Reply by funkywaddle** [reply]
I think I did the same as the OP in that post and installed grunt-cli via apt and not npm. Once I did the npm installs, everything seemed to work.

Thanks!

**Comment by marcelfolaron** [comment]
Perfect!

**Comment by marcelfolaron** [comment]
Odd, I’ve never seen this issue before but seems a module is missing. Did a
quick search: can you try to follow the suggestions here:
https://stackoverflow.com/questions/53408846/cannot-run-grunt-command-because-of-error-cannot-find-module-liftoff
